source/web-service/flaskapp/routes/sparql.py

```diff
@@ -72,8 +75,14 @@ def query_entrypoint():
         accept_header = request.headers.get("Accept")
 
     query_endpoint = current_app.config["SPARQL_QUERY_ENDPOINT"]
+
     if request.method == "POST":
-        res = execute_query_post(request.form, accept_header, query_endpoint)
+        res = execute_query_post(
```

Comment:
Just a small comment needed to remind anyone reading this that the dict call is to handle POST requests with the query in a URL-encoded parameter instead of the form. Otherwise, that mix of iterable and kwarg dict instantiation is a little unusual and what it does may not be obvious at first.